src/floating-sdk-imweb.css

```
@@ -527,4 +527,65 @@
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.guest-access-block-md {
+    height: calc(100% - 44px);
+}
+
+.guest-access-block-title {
+    font-family: "pretendard";
```

Stylelint (17.10.0)

[error] Expected no quotes around "pretendard" (font-family-name-quotes)

(font-family-name-quotes)